R/get_analysis_date.R

```diff
@@ -19,14 +19,14 @@
 
 #' Derive analysis date for interim/final analysis given multiple conditions.
 #'
-#' To obtain the analysis date, consider the following multiple conditions:
```

Comment:
I liked this introductory line. It's a bit jarring to get dropped directly into the list of multiple conditions

Reply:
I agree. This line is now restored before the list.